frontend/src/components/Transactions/TransactionLinksSummary.vue

```diff
@@ -0,0 +1,125 @@
+<template>
+  <div class="transaction-slot-link">
+    <div>
+      <div @click="visible = !visible">
+        <!-- Collaps Icon  -->
+        <collapse-icon class="text-right" :visible="visible" />
+        <div>
+          <b-row>
+            <!-- ICON  -->
+            <b-col cols="1">
+              <type-icon color="text-danger" icon="link45deg" />
+            </b-col>
+
+            <b-col cols="11">
+              <!-- Amount / Name || Text -->
+              <amount-and-name-row :amount="amount" :text="$t('gdd_per_link.links_sum')" />
+
+              <!-- Count Links -->
+              <link-count-row :count="transactionLinkCount" />
+
+              <!-- Decay -->
+              <decay-row :decay="decay" />
+            </b-col>
+          </b-row>
+        </div>
+      </div>
+
+      <b-collapse v-model="visible">
+        <collapse-links-list
+          v-model="variables"
+          :transactionLinkCount="transactionLinkCount"
+          :transactionLinks="transactionLinks"
+        />
+      </b-collapse>
+    </div>
+  </div>
+</template>
+<script>
+import CollapseIcon from '../TransactionRows/CollapseIcon'
+import TypeIcon from '../TransactionRows/TypeIcon'
+import AmountAndNameRow from '../TransactionRows/AmountAndNameRow'
+import LinkCountRow from '../TransactionRows/LinkCountRow'
+import DecayRow from '../TransactionRows/DecayRow'
+import CollapseLinksList from '../DecayInformations/CollapseLinksList'
+import { listTransactionLinks } from '@/graphql/queries'
+
+export default {
+  name: 'TransactionSlotLink',
+  components: {
+    CollapseIcon,
+    TypeIcon,
+    AmountAndNameRow,
+    LinkCountRow,
+    DecayRow,
+    CollapseLinksList,
+  },
+  props: {
+    amount: {
+      type: String,
+      required: true,
+    },
+    decay: {
+      type: Object,
+      required: true,
+    },
+    transactionLinkCount: {
+      type: Number,
+      required: true,
+    },
+  },
+  data() {
+    return {
+      visible: false,
+      transactionLinks: [],
+      variables: {
+        currentPage: 1,
+        pending: false,
+        pageSize: 5,
+        itemsShown: 0,
+      },
```

Comment:
```suggestion
    return {
      visible: false,
      transactionLinks: [],
      pending: false,
      variables: {
        currentPage: 1,
        pageSize: 5,
        itemsShown: 0,
      },
```

Reply:
pending muss in variables drin bleiben .. da variables im v-model übergeben wird und in einer anderen componente mit value verwendet wird.